src/rtc/mosaic_geobursts.py

```diff
@@ -325,9 +532,9 @@ def compute_weighted_mosaic_raster(list_rtc_images, list_nlooks, geo_filename,
 
```

Comment:
```suggestion
```

Another nit pick

Reply:
Thank you, @seongsujeong . I'm trying to use about 2 lines between functions.